gendiff/scripts/gendiff.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import argparse
+
+
+def generate_help_info() -> None:
+    """Create CLI help for program running command"""
+
+    parser = argparse.ArgumentParser(
+        prog='gendiff',
+        description='Compares two configuration files and shows a difference.')
+    parser.add_argument('first_file')
+    parser.add_argument('second_file')
+    args = parser.parse_args()
+    print(parser, args)
+
+
+def main():
+    generate_help_info()
```

Comment:
[optional] I'm not sure if this function will be used anywhere else, can be inlined